include/deal.II/lac/affine_constraints.templates.h

```diff
@@ -130,7 +130,7 @@ AffineConstraints<number>::is_consistent_in_parallel(
     if (line_index >= lines_cache.size() ||
         lines_cache[line_index] == numbers::invalid_size_type)
       {
-        const ConstraintLine empty = {line_n, {}, 0.0};
+        ConstraintLine empty = {line_n, {}, 0.0};
```

Comment:
How about this instead:
```suggestion
        return ConstraintLine{line_n, {}, 0.0};
```

Reply:
Sure, I would always prefer RVO if we can.